aiida/backends/tests/cmdline/commands/test_export.py

```diff
@@ -68,6 +70,9 @@ def setUpClass(cls, *args, **kwargs):
 
         # Utility helper
         cls.fixture_archive = 'export/migrate'
+        cls.newest_archive = "export_v{}_simple.aiida".format(EXPORT_VERSION)
+        cls.penultimate_archive = "export_v{}_simple.aiida".format(
+            round(float(EXPORT_VERSION) - 0.1, len(EXPORT_VERSION) - len("0.")))
```

Comment:
Yeah, I know.
If you have any suggestions here, they are more than welcome!
I tried making it general and nice, but the floating point numbers made it quite difficult to make it precise (e.g. hitting 0.799999999 instead of 0.8).
I could just go back to a more hard-coded way of doing it?

Reply:
Have now hardcoded it (it is not the most important thing to remember to update, luckily).
Furthermore, I have removed the same logic that was present in the test `test_wrong_versions` in `aiida.backends.tests.tools.importexport.migration.test_migration.TestExportFileMIgration`. Here it was supposed to add 0.1 to the current export version. Instead, I simply added 0.99.